crates/rpc-types-anvil/src/lib.rs

```diff
@@ -148,6 +148,7 @@ pub enum MineOptions {
         /// The timestamp the block should be mined with
         #[serde(with = "alloy_serde::u64_opt_via_ruint")]
         timestamp: Option<u64>,
+        #[serde(with = "alloy_serde::u64_opt_via_ruint")]
```

Comment:
Move to below doc-comments

Reply:
Opted to remove this addition for now, going to handle any changes to the types in https://github.com/alloy-rs/alloy/issues/838